README.md

```diff
@@ -24,8 +24,9 @@ Our [PyPI wheels](https://pypi.org/project/libzim/) bundle a [recent release](ht
 - macOS for `x86_64` and `arm64`
 - GNU/Linux for `x86_64`, `armhf` and `aarch64`
 - Linux+musl for `x86_64` and `aarch64`
+- Windows for `x64`
 
-Wheels are available for both CPython and PyPy.
+Wheels are available for both CPython only  (but can be built for Pypy).
```

Comment:
extra whitespace in `only  (but` => `only (but`

Reply:
`for both CPython only` => `for CPython only`

Reply:
Indeed!